sdk/webpubsub/Azure.Messaging.WebPubSub/api/Azure.Messaging.WebPubSub.netstandard2.0.cs

```diff
@@ -5,6 +5,11 @@ public static partial class ClientConnectionFilter
         public static string Create(System.FormattableString filter) { throw null; }
         public static string Create(System.FormattableString filter, System.IFormatProvider formatProvider) { throw null; }
     }
+    public enum ClientType
+    {
+        Default = 0,
+        MQTT = 1,
```

Comment:
[Acronyms get PascalCased in .NET](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions#capitalization-rules-for-identifiers) so this should be [Mqtt](https://learn.microsoft.com/en-us/dotnet/api/?term=mqtt).